modin/pandas/__init__.py

```diff
@@ -12,6 +12,14 @@
 # governing permissions and limitations under the License.
 
 import pandas
+from enum import Enum
+
+
+class Axis(Enum):
+    ROW_WISE = 0
+    COL_WISE = 1
+    CELL_WISE = None
```

Comment:
Could you explain why the member_value for CELL_WISE = None instead of 2, -1? It probably doesn't matter since we're mostly going to use this to do `if axis == Axis.CELL_WISE:` anyways.

Side note: semantically an axis is a line of rotation, which doesn't make as much sense with CELL_WISE. Ofc it's probably still better to have CELL_WISE here since without it we'd need to introduce another param or rename axis.

Reply:
That's a good point - I can think of two reasons to keep I `None` off the top of my head, but I'm happy to change it if you think another value would work better!
1. I wanted to minimize the (breaking) changes we make. Cell-wise has been `None` so far, so if we were to change it to a different value, we'd need to reflect the change not just here, but also across all of the places where these algebra operators are called. 
2. This goes hand in hand with your second point - cell-wise technically isn't an axis, like you mentioned. Instead it's kind of the absence of an axis - instead of traversing along the horizontal or vertical axis, we're traversing along partitions or cells. In fact, cell wise isn't even truly cell-wise - it's actually partition-wise. When we spoke about the axes, I usually say explain cell-wise as parallel across cells because that's an easy abstraction for me to remember, and is technically true - but in practice, we're iterating over partitions, and passing chunks of the df to the function at a time.

Reply:
We could potentially do `Axis.NO_AXIS`? And also rename to `Axis.ROW_AXIS`, etc.; although I feel like row axis is less clear than row-wise in my opinion.

Reply:
Thanks for the explanation! I agree NO_AXIS and ROW_AXIS don't make as much sense as what you have now. I don't have any further thoughts on this, so I will defer to you and Devin.